clients/web/src/views/widgets/UploadWidget.js

```diff
@@ -173,8 +173,8 @@ girder.views.UploadWidget = girder.View.extend({
     startUpload: function (e) {
         e.preventDefault();
 
+        this.setUploadEnabled(false);
         this.$('.g-drop-zone').addClass('hide');
-        this.$('.g-start-upload').addClass('disabled');
```

Comment:
I like to group together lines that look similar. Don't judge me. :stuck_out_tongue: